courseProject/src/test/java/com/neoflex/CourseProject/service/ImplPreImplScoringServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package com.neoflex.CourseProject.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.math.BigDecimal;
+
+@ExtendWith(MockitoExtension.class)
+class ImplPreImplScoringServiceTest {
```

Comment:
Во-первых, что-то очень намудрил с названием тест класса. "Impl" суффикс является сокращением от слова "implementation", которое ты используешь для имплементации интерфейсов в своих классах. Соответственно, в данном тест классе ты ничего не имплементируешь, зачем тут Impl? И все-таки мы тестируем интерфейсы, а не сами сервисы, следует изменить название;
Во-вторых, добавь над каждым тест классом и методами @DisplayName для отображения названия и дополнительной информации в консоли в зависимости от того, что они должны проверять.
```suggestion
@DisplayName("Мок-тестирование метода сервиса PrescoringService")
class PrescoringServiceTest {
```

Reply:
так?
![image](https://github.com/user-attachments/assets/a95579ba-bff4-42f0-b107-bce2f605a180)